pkg/controller/podgc/gc_controller_test.go

```diff
@@ -749,12 +747,12 @@ func TestGCInspectingPatchedPodBeforeDeletion(t *testing.T) {
 
 			actions := client.Actions()
 
-			var patchAction clienttesting.PatchAction
+			var patchAction clienttesting.UpdateAction
```

Comment:
```suggestion
			var updateAction clienttesting.UpdateAction
```

Reply:
nit: not sure if we want to update the name of the test as well..

Reply:
I would say that its ok in the test. I think one could say update is a way to patch the pod here :)

Reply:
I'm inclined to leave patchedPodNames also, because this would affect many tests, and I think it is better to keep the fix concise as it is intended for cherry-picking.